tests/integration/security/sds_vault_flow/sds_vault_flow_test.go

```diff
@@ -109,15 +81,15 @@ func TestSdsVaultCaFlow(t *testing.T) {
 	namespace := istioCfg.SystemNamespace
 	configFile := "global-mtls.yaml"
 
-	policy := applyPolicyFile(t, env, namespace, configFile)
+	policy := policy.ApplyPolicyFile(t, env, namespace, configFile)
```

Comment:
I can't tell the different of this test vs the other reach-ability one reading the test code (my best guess is smt hidden in the config yaml). Anyway, this is the same point I mentioned above. If the test mechanism for these are the same (i.e setup some policy, then verify connectivity), they probably should be in the same test suite (folder).

Reply:
I think organizing tests into folders based on the features, as opposed to dividing them based on different frameworks (i.e., kube vs native) are easier to locate the test of a feature.

Reply:
I suggested group 'thing' to verify, e.g connectivity in this case (test framework is artifact, as some test are bound by them). If you divide by (security) features, they will be scatter in too much, and worst, you soon end up with test cases that use more than one feature (say rbac + jwt + mtls + sds + vault) and so on. Think of a folder like a test suite. If the test cases are different lightly on the input data (e.g runtime configs), it would be nicer that they are together.

Reply:
Let's continue this discussion offline and after that we can revise the folder organization in a separate PR.